examples/multi_inspiral/faceon_faceaway.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -e
+
+EVENT=1187008882
+STEP=300
+START=$((EVENT - 2 * STEP))
+END=$((EVENT + STEP + 64))
+GPS_START=$((START - 32))
+GPS_END=$((END + 32))
+DUR=$((GPS_END - GPS_START))
+CONFIG_URL=https://github.com/gwastro/pycbc-config/raw/master/test/multi_inspiral
+CONFIG_URL=https://github.com/a-r-williamson/pycbc-config/raw/pycbc_multi_inspiral_tests/test/multi_inspiral
+INJ_FILE=faceon_faceaway_injs.hdf
+
+echo -e "\\n\\n>> [`date`] Getting injection file"
+wget -nc ${CONFIG_URL}/${INJ_FILE}
+
+# Generate mock data
+for IFO in 'H1' 'L1' 'V1'; do
+    echo -e "\\n\\n>> [`date`] generating mock data for ${IFO}"
+    `which pycbc_condition_strain` \
```

Comment:
Why is the `which` needed?

Reply:
I had just followed the lead of [examples/inspiral/run.sh](https://github.com/gwastro/pycbc/blob/master/examples/inspiral/run.sh) which evaluates the path with `which` but I guess there's no need.

Reply:
Ah I see, in that case it was needed because `pycbc_inspiral` was being run through the cProfile module (which has then been commented out). You can just call `pycbc_condition_strain` here.